### PR TITLE
Home page style added

### PR DESCRIPTION
**Pull request by Marirood**
Now the home page has an introduction to CS System and there are three buttons as needed.

### CODE REVIEW

File: src/CrystallineSociety/Client/Shared/Pages/HomePage.razor.scss [hunk 1]
```diff
@@ -2,103 +2,11 @@
...
-    justify-content: flex-start;
-    align-items: center;
-    width: 100%;
-    padding: 0 rem(16px);
-    box-sizing: border-box;
-}
-
-.page-section {
+.home-actions {
     width: 100%;
```
Comment (hootanht) [inline src/CrystallineSociety/Client/Shared/Pages/HomePage.razor.scss:6]: We have this attribute in Bootstrap called `w-100`

File: src/CrystallineSociety/Client/Shared/Pages/HomePage.razor.scss [hunk 2]
```diff
@@ -2,103 +2,11 @@
...
-}
-
-.hero-section {
     display: flex;
```
Comment (hootanht) [inline src/CrystallineSociety/Client/Shared/Pages/HomePage.razor.scss:7]: For display and other attributes we have class name in Bootstrap like `d-flex`
@Marirood

File: src/CrystallineSociety/Client/Shared/Shared/Header.razor.scss [hunk 3]
```diff
@@ -2,59 +2,6 @@
...
-        i {
-            color: $PrimaryColor;
-        }
-    }
-
-    @include lt-lg {
-        display: block;
-    }
+.user-box {
+    text-align: right;
```
Comment (hootanht) [inline src/CrystallineSociety/Client/Shared/Shared/Header.razor.scss:6]: We have this attribute class name on Bootstrap

File: src/CrystallineSociety/Client/Web/Pages/_Layout.cshtml [hunk 4]
```diff
@@ -49,11 +49,18 @@
...
+            margin: 0;
+            padding: 0;
+            font-size: 62.5%;
+        }
+
+        body, #app-container {
             margin: 0;
             padding: 0;
             width: 100%;
-            height: 100%;
```
Comment (hootanht) [inline src/CrystallineSociety/Client/Web/Pages/_Layout.cshtml:56]: We must use container of Bootstrap

File: src/CrystallineSociety/Client/Shared/Pages/HomePage.razor [hunk 5]
```diff
@@ -3,44 +3,16 @@
 
 <PageTitle>@Localizer[nameof(AppStrings.HomeTitle)]</PageTitle>
 
-<div class="page-container">
-    <section class="page-section hero-section">
-        <h1 class="hero-section-title">@Localizer[nameof(AppStrings.HomeTitle)]</h1>
+<div class="container">
+    <h1 class="h1">CS System</h1>
```
Comment (hootanht) [inline src/CrystallineSociety/Client/Shared/Pages/HomePage.razor:7]: We must use resources for our text

File: src/CrystallineSociety/Client/Shared/Pages/HomePage.razor.scss [hunk 6]
```diff
@@ -2,103 +2,11 @@
...
-    height: rem(34px);
-    border: rem(1px) solid $White;
-    color: $White;
-    background-color: $SecondaryColor;
-
-    &:hover {
-        background-color: $SecondaryColor;
-        border-color: $White;
-    }
+    gap: 1rem;
```
Comment (hootanht) [inline src/CrystallineSociety/Client/Shared/Pages/HomePage.razor.scss:11]: I think we have class name in Bootstrap for gap

File: src/CrystallineSociety/Client/Shared/Shared/MainLayout.razor [hunk 7]
```diff
@@ -2,6 +2,20 @@
 @inherits LayoutComponentBase
```
Comment (hootanht) [inline src/CrystallineSociety/Client/Shared/Shared/MainLayout.razor:2]: Why this inherits deleted?

File: src/CrystallineSociety/Client/Shared/Shared/MainLayout.razor.scss [hunk 8]
```diff
@@ -2,178 +2,19 @@
...
-
-    @include sm {
-        max-width: 100%;
-    }
-}
-
-::deep .form-submit-button {
-    margin-top: rem(20px);
+    justify-content: flex-start;
+    align-items: stretch;
```
Comment (hootanht) [inline src/CrystallineSociety/Client/Shared/Shared/MainLayout.razor.scss:14]: We have this on Bootstrap

**Review by hootanht — CHANGES_REQUESTED** [review]

File: src/CrystallineSociety/Client/Shared/Pages/HomePage.razor.scss (hunk 1 above)
Comment (Marirood) [reply src/CrystallineSociety/Client/Shared/Pages/HomePage.razor.scss]: Done

File: src/CrystallineSociety/Client/Web/Pages/_Layout.cshtml (hunk 4 above)
Comment (Marirood) [reply src/CrystallineSociety/Client/Web/Pages/_Layout.cshtml:56]: This is in _Layout.cshtml. It's better to be like this.

File: src/CrystallineSociety/Client/Shared/Pages/HomePage.razor.scss (hunk 2 above)
Comment (Marirood) [reply src/CrystallineSociety/Client/Shared/Pages/HomePage.razor.scss]: done.

File: src/CrystallineSociety/Client/Shared/Shared/Header.razor.scss (hunk 3 above)
Comment (Marirood) [reply src/CrystallineSociety/Client/Shared/Shared/Header.razor.scss:6]: done

**Pull request merged**